cheatsheet.md

```diff
@@ -84,11 +89,17 @@ property is set to `""`. Note that `skip-activation` implies
   selected as well (disabled by `skip-viewport-activation`)
 * focus(): when focus() is called on an element, its locked ancestors are
   activated (disabled by `skip-activation`)
+
 ## Current status (Chromium)
 
-* **Activation**: As of [this patch](https://chromium-review.googlesource.com/c/chromium/src/+/1853854),
-  activation is the default behavior, which can be disabled with
-  `skip-activation` or `skip-viewport-activation`. A `beforeactivate` event will be fired before showing the new content on-screen.
+* **Activation**: Activation is the default behavior, which can be disabled with
+  `skip-activation` or `skip-viewport-activation`. A `beforeactivate` event will
+  be fired before showing the new content on-screen.
```

Comment:
This event behavior is not true - we instead post a task.

Reply:
I think we just fire it: https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/display_lock/display_lock_context.cc?l=384

There is a posttask to deliver the intersection observation, is this what you mean? Most activations do it right as we're activating. We are going to change this based on our previous discussions, but as it stands now I think the text is more or less correct. WDYT?

Reply:
We shouldn't fire sychronously during C++ execution, or there will be re-entrancy issues...